crates/provider/src/native.rs

```diff
@@ -29,19 +29,18 @@ use crate::{
 };
 #[derive(Debug, Serialize, Clone, PartialEq)]
 pub struct NativeProvider<T: FileSystem + Send + Sync> {
-    // Namespace of the client
+    /// Namespace of the client (isolation directory)
     namespace: String,
-    // Path where configuration relies
+    /// Path where configuration relies, all the `files` are accessed relative to this.
     config_path: String,
-    // Variable that shows if debug is activated
+    /// Variable that shows if debug is activated
     is_debug: bool,
-    // The timeout for the client to exit
+    /// The timeout for start the node
     timeout: u32,
-    // Command sent to client
+    /// Command to use, e.g "bash"
     command: String,
-    // Temporary directory
+    /// Temporary directory, root directory for the network
     tmp_dir: String,
```

Comment:
All these comments won't be visible, fields are private, you can check it run ```cargo doc --open``` in your terminal